packages/vscode-extension/package.nls.json

```diff
@@ -37,5 +37,227 @@
     "teamstoolkit.commands.environmentsLink.title": "Learn more about enviornments",
     "teamstoolkit.commands.developmentLink.title": "Learn more about development",
     "teamstoolkit.commands.deploymentLink.title": "Learn more about deployment",
-    "teamstoolkit.commands.feedbackLink.title": "Learn more about help and feedback"
-}
\ No newline at end of file
+    "teamstoolkit.commands.feedbackLink.title": "Learn more about help and feedback",
+    "teamstoolkit.common.activate": "Teams Toolkit extension is now active!",
+    "teamstoolkit.common.userCancel": "User Cancel",
+    "teamstoolkit.common.confirm": "Confirm",
+    "teamstoolkit.common.signout": "Sign out",
+    "teamstoolkit.common.signOutOf": "Sign out of '%s'?",
+    "teamstoolkit.common.cancel": "Cancel",
+    "teamstoolkit.common.signin": "Sign in",
+    "teamstoolkit.common.readMore": "Read more",
+    "teamstoolkit.userInterface.noQuestionDescription": "No question description",
```

Comment:
How about considering alpha-bate order for easier maintenance?

Reply:
Do you mean the way as Strings.json? It's required format for VSC extension localization. The old way doesn't work.

Reply:
(My bad typo...)
I mean the alphabetic order for all keys, e.g.,
```
"teamstoolkit.a....",
...
"teamstoolkit.b...",
...
"teamstoolkit.c...",
...
```

Reply:
Good suggestion!

Reply:
Updated